flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/source/datagen/SequenceGenerator.java

```diff
@@ -83,9 +86,11 @@ public void open(
             final int taskIdx = runtimeContext.getIndexOfThisSubtask();
             final long congruence = start + taskIdx;
 
+            Preconditions.checkArgument(
+                    end - start < Long.MAX_VALUE - 1, "Limit exceeded Long.MAX_VALUE-1");
```

Comment:
```suggestion
                    end - start < Long.MAX_VALUE - 1, "Limit exceeded " + Long.MAX_VALUE - 1);
```
Don't we want to print the actual value here? I'm a bit torn here because I see the problem with just printing a really big number. Going for `Long.MAX_VALUE` might actually more readabile. WDYT?

Reply:
yes, I think it would be awkward to print a large number, outputting Long.MAX_VALUE may be easier for people to understand